ui/app/src/utils/path.ts

```diff
@@ -41,6 +41,18 @@ export function getPathFromPreviewURL(previewURL: string) {
   return `/site/website${pagePath}`;
 }
 
+export function isEditableFormAsset (path: string) {
+  return path.indexOf(".ftl") != -1
```

Comment:
Use `includes` instead of `indexOf`. But better yet, compare with mimeType. We might want to reconsider where to locate this util - not sure this is the most semantic place for it. 🤔